src/main/java/chess/controller/GameController.java

```diff
@@ -0,0 +1,62 @@
+package chess.controller;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import chess.dto.BoardDto;
+import chess.dto.MoveDto;
+import chess.dto.ResultDto;
+import chess.service.GameService;
+
+@RestController
+@RequestMapping("/game")
+public class GameController {
+
+    private final GameService gameService;
+
+    public GameController(GameService gameService) {
+        this.gameService = gameService;
+    }
+
+    @PostMapping("/{id}/start")
+    public BoardDto start(@PathVariable Long id) {
+        return gameService.start(id);
+    }
+
+    @PostMapping("/{id}/end")
+    public BoardDto end(@PathVariable Long id) {
+        return gameService.end(id);
+    }
+
+    @GetMapping("/{id}/status")
+    public ResultDto status(@PathVariable Long id) {
+        return gameService.status(id);
+    }
+
+    @GetMapping("/{id}/load")
+    public BoardDto load(@PathVariable Long id) {
+        return gameService.load(id);
+    }
+
+    @PostMapping("/{id}/move")
+    public BoardDto move(@PathVariable Long id,
+        @RequestBody MoveDto moveDto) {
+        return gameService.move(id, moveDto);
+    }
+
+    @ExceptionHandler(RuntimeException.class)
+    public ResponseEntity<Object> handle(RuntimeException ex) {
+        Map<String, String> error = new HashMap<>();
+        error.put("message", ex.getMessage());
+        return ResponseEntity.badRequest().body(error);
```

Comment:
```suggestion
    public ResponseEntity<Map<String, String>> handle(RuntimeException ex) {
        Map<String, String> error = new HashMap<>();
        error.put("message", ex.getMessage());
        return ResponseEntity.badRequest().body(error);
```

경우에 따라 조금씩 다르겠지만 타입 파라미터에는 Object가 들어갈 일이 많지는 않은 것 같아요. 이렇게 해두고 보니 DTO로 감싸는게 조금 더 좋아보이겠다는 생각도 드네요.